### PR TITLE
Adding project section

### PR DESCRIPTION
**Pull request by houdamzari**
- Added Grid Layout 
- Added Flexbox to align the cards.
- Used lists on navigation links .
- Implemented buttons interactions ( on hover, on press )
- Added project and about section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,78 +11,243 @@
...
+            <div>Javascript</div>
+          </div>
+          <div class="container">
+            <img src="./public/Html_elipse.png" alt="html" />
+            <div>Html</div>
+          </div>
+          <div class="container">
+            <img src="./public/Css_ellipse.png" alt="css" />
+            <div>CSS</div>
+          </div>
```
Comment (Lizdev-05) [inline index.html]: Good job so far @houdamzari☺, kindly wrap the technologies in this section i.e **JavaScript, CSS, and Html** inside of the `ul` tag in accordance with the requirement.

File: index.html [hunk 2]
```diff
@@ -11,78 +11,243 @@
...
+      </div>
+    </section>
+    <section class="About_Section">
+      <h1>About <br />Myself</h1>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="links-text">Let's connect</div>
```
Comment (Lizdev-05) [inline index.html:198]: You have a great and neat code here 👍, but Kindly wrap 'Let's connect' inside a `header` tag to improve the scalability of your code.

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there-reach

## Highlights
- Correct GitHub Flow 👍🏾
- Good commit history 🥇
- Good PR title/description 💯
- All linters passed 🥇
- Personalized README file ✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:
- Before submitting your work for review in future projects, kindly close unused pull requests to avoid confusion.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @houdamzari , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**